devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy

```diff
@@ -94,24 +95,32 @@ def updated() {
 def getCommandClassVersions() {
 	[
 		0x20: 1,  // Basic
-		0x32: 1,  // SwitchMultilevel
+		0x32: 3,  // Meter
```

Comment:
you might want to have the version of meter used be conditional based on the model of the device in order to preserve legacy functionality.

Based on this, all meter reports will be parsed as v3 so you will not receive any meterv1 events.